site/gatsby-site/page-creators/createCitiationPages.js

```diff
@@ -49,172 +50,180 @@ const getClassificationsArray = (incidentClassifications, taxonomy) => {
   return array;
 };
 
-const createCitiationPages = (graphql, createPage) => {
-  return new Promise((resolve, reject) => {
-    resolve(
-      graphql(
-        `
-          query IncidentIDs {
-            allMongodbAiidprodIncidents {
-              distinct(field: incident_id)
-              group(field: incident_id) {
-                fieldValue
-                edges {
-                  node {
-                    id
-                    submitters
-                    incident_date
-                    date_published
-                    incident_id
-                    report_number
-                    title
-                    url
-                    image_url
-                    cloudinary_id
-                    source_domain
-                    mongodb_id
-                    text
-                    authors
-                  }
-                }
+const createCitiationPages = async (graphql, createPage) => {
+  const result = await graphql(
+    `
+      query IncidentIDs {
+        allMongodbAiidprodIncidents(filter: { incident_id: { ne: 0 } }) {
```

Comment:
Why `ne: 0`?

Reply:
that's to ignore a "dummy content" incident. It is probably a leftover, I'll delete it